src/server/worker/datum/datum.go

```diff
@@ -258,6 +258,9 @@ func (d *Datum) downloadData(downloader pfssync.Downloader) error {
 				return nil
 			}),
 		}
+		if input.S3 {
```

Comment:
nit, would you move this above the opts setup.

Reply:
Done.